### DISCUSSION TITLE
Suggestion: Google Ads to Support Scratch

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#4741] [Ideas] [answered]
I was wanting to support scratch, but I don't have a good way to donate money. I thought it would be a cool idea to have the ability to turn on google adverts that can be spread throughout the site. I'm not sure if this would violate scratch's privacy policy because Google Ads would track you, but I don't know. I think this would be a cool idea, and sites like monkeytype.com already use this feature. I want to support scratch.

**Comment by Samq64** [comment, marked as answer]
Scratch is a non-profit organization, so they can't do paid ads. See 7.7 on Scratch's list of rejected suggestions. (https://scratch.mit.edu/discuss/topic/343602/) As an addon it _might_ by OK but I don't think anyone would enable it (because who would opt-in to ads that track you) and it might display inappropriate ads sometimes.

> I'm not sure if this would violate scratch's privacy policy because Google Ads would track you

Scratch already uses Google Analytics, so privacy wouldn't be an issue.

**Reply by bobby-sills** [reply]
Yeah, and now that I read the rejected suggestion even more, they say:

> and that would be a huge legal issue.

And I don't want to get the scratch website in any legal trouble

**Reply by BroJac5246** [reply]
Technically we _could_ implement this, since the user would basically be creating their own revenue and donating it. But why? Does the Scratch Foundation even need the revenue?

**Reply by WorldLanguages** [reply]
Chrome Web Store would kick us right away if we did this

**Reply by bobby-sills** [reply]
Why?

**Reply by WorldLanguages** [reply]
Single purpose policy

**Comment by rimopa** [comment]
You can donate to the scratch foundation [here](https://secure.donationpay.org/scratchfoundation/), if you want to do it.

**Comment by Hans5958** [comment]
You mean, you are asking us to insert our Google Ads (we are the publisher) to the website, and somehow trust us to donate everything to Scratch Foundation and not embezzle it?

**Reply by BroJac5246** [reply]
Yup
